delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/routes/ElasticSearchViewsRoutes.scala

```diff
@@ -104,17 +104,17 @@ final class ElasticSearchViewsRoutes(
                               (parameter("rev".as[Long].?) & pathEndOrSingleSlash & entity(as[Json])) {
                                 case (None, source)      =>
                                   // Create an elasticsearch view with id segment
-                                  emit(Created, views.create(id, ref, source).map(_.void))
+                                  emit(Created, views.create(id, ref, source).map(_.void).rejectOn[DecodingFailed])
                                 case (Some(rev), source) =>
                                   // Update a view
-                                  emit(views.update(id, ref, rev, source).map(_.void))
+                                  emit(views.update(id, ref, rev, source).map(_.void).rejectOn[DecodingFailed])
```

Comment:
what about `ViewNotFound`?

Reply:
`DecodingFailed` happens before `ViewNotFound`

Reply:
Yes, but It doesn't hurt to add that too, right? Since we could change the order of rejections and many things will keep working